### ISSUE TITLE
No.

### ISSUE THREAD

**Issue by danitt** [joelshepherd/render#1]
Issue: this exists.
Solution: **no.**

**Comment by danitt** [comment]
I agree with @danitt 's idea, we should try implement **no.** and see if it resolves the issue.

**Comment by danitt** [comment]
I have totally changed my stance on this, and am now massively in support of the ongoing development of this project - @joelshepherd where's that **sponsor** button at??

**Comment by alanmaq** [comment]
I'm with @danitt .  More rendering please.

**Comment by joelshepherd** [comment]
Don't forget to like & subscribe to my other rendering experiments, but this time, _in wasm_.

https://github.com/joelshepherd/ui-rs/blob/main/example/main.js
